crates/relayer/src/chain/ckb/utils.rs

```diff
@@ -85,9 +85,9 @@ where
 
 pub fn align_native_and_onchain_updates<S, E>(
     chain_id: &str,
-    header_updates: &Vec<EthUpdate>,
+    header_updates: &mut Vec<EthUpdate>,
```

Comment:
If I remembered correctly, `clippy` will throw errors.
- `&Vec<T>` -> `&[T]`
- `&mut Vec<T>` -> `&mut [T]`

Reply:
no error throw, maybe clippy could pass for the mutable use